Leanplum/Api/LPWSManager.h

```diff
@@ -11,7 +11,7 @@
 
 @interface LPWSManager : NSObject
 
-- (void)sendGETWebService:(NSString*)service userParams:(NSMutableDictionary *)userParams successBlock:(void (^)(NSDictionary *))success failureBlock:(void (^)(NSError *))failure;
-- (void)sendPOSTWebService:(NSString*)service userParams:(NSMutableDictionary *)userParams successBlock:(void (^)(NSDictionary *))success failureBlock:(void (^)(NSError *))failure;
+- (void)sendGETWebService:(NSString*)service userParams:(NSDictionary *)userParams successBlock:(void (^)(NSDictionary *))success failureBlock:(void (^)(NSError *))failure;
```

Comment:
this doesn't have to specifically be user related params right? why don't we just name it params?